src/redux/data.ts

```diff
@@ -0,0 +1,40 @@
+export interface W2Information {
+  SSID: string
+  employeeFirstName: string
```

Comment:
Makes sense. My next step was going to be to add taxpayer + spose + dependent information. So this W2employee data will probably go away, and all that will be required is an indication of which spouse received the W2. I think the coupling is apparent now because the data represented is so simple and one-to-one. Soon this will change.

Reply:
Makes sense. My next step was going to be to add taxpayer + spose + dependent information. So this W2employee data will probably go away, and all that will be required is an indication of which spouse received the W2. I think the coupling is apparent now because the data represented is so simple and one-to-one. Soon this will change.